src/verify/index.ts

```diff
@@ -1,7 +1,18 @@
 import { timingSafeEqual } from "crypto";
 import { Buffer } from "buffer";
+import { EventTypesPayload } from "../generated/get-webhook-payload-type-from-event";
 import { sign } from "../sign/index";
 
+type WebhookEvents = Exclude<
+  keyof EventTypesPayload,
+  `${string}.${string}` | "errors" | "*"
```

Comment:
woah I didn't know you can do the template literals with `string` types! Very cool

Reply:
Yeah, thats the nice hotness that landed with 4.1 - it only arrived about 2 weeks ago.

I'm actually very happy with this bit of code, because it's my first sensible reason to use it in production code, and it works like a treat 😍